features/all-positions/PositionActionsDialog.tsx

```diff
@@ -283,6 +284,19 @@ const PositionActionsDialog = (props: DialogProps) => {
       }
     };
 
+    useEffect(() => {
+      // Set min and max collPerToken params for the Liquidation dialog to
+      // approximately the current CRatio, +/- some error threshold.
+      const collPerToken =
+        Number(sponsorPosition.backingCollateral) /
+        Number(sponsorPosition.tokensOutstanding);
+      setMinCollPerToken((collPerToken * 0.95).toFixed(10));
+      setMaxCollPerToken((collPerToken * 1.05).toFixed(10));
```

Comment:
nit: I think these are strange defaults, but they might be _good enough_.

Intuitively, the min collateral should be the point at which you'll lose money on the liquidation (i.e. the total collateral in the position should be > 100% + some cushion) and the max collateral should be the point at which the liquidation becomes invalid (i.e. the total collateral - withdraw request should be < 125% - some cushion). Do you think it would be hard to compute those values? If so, it may not be worth it, but just wanted to mention it.

Reply:
This could be possible to calculate but currently we do not even block liquidations if they would be unprofitable. Perhaps we change this in a separate PR @chrismaree wdyt?

Reply:
How would you calculate the smart min and max's? I agree that those should be the thresholds

Reply:
Implemented actually pretty easy to calculate. See comments for details